src/lib/operators/join_sort_merge/radix_cluster_sort.hpp

```diff
@@ -66,6 +68,17 @@ class RadixClusterSort {
 
   virtual ~RadixClusterSort() = default;
 
+  template <typename T2>
+  static std::enable_if_t<std::is_integral_v<T2>, size_t> get_radix(T2 value, size_t radix_bitmask) {
+    return static_cast<int64_t>(value) & radix_bitmask;
+  }
+
+  template <typename T2>
+  static std::enable_if_t<!std::is_integral_v<T2>, size_t> get_radix(T2 value, size_t radix_bitmask) {
```

Comment:
`is_floating_point`?

Reply:
And function for `pmr_string` appears to be missing?

Reply:
since float and string now both use std::hash, there is no need anymore for dedicated pmr_string/is_floating_point impls. they just share this one

Reply:
🤦